package.json

```diff
@@ -30,8 +30,9 @@
   "license": "MIT",
   "dependencies": {
     "coffee-script-redux": "https://github.com/michaelficarra/CoffeeScriptRedux.git",
+    "detect-indent": "^4.0.0",
     "magic-string": "^0.6.5",
-    "repeat-string": "^1.5.2"
+    "repeating": "^2.0.0"
```

Comment:
`repeating` is a dependency of `detect-indent`, so we're already bundling it. Might as well consolidate!